_partials/_cloud_self_configuration.md

```diff
@@ -0,0 +1,87 @@
+import EarlyAccess from "versionContent/_partials/_early_access.mdx";
+
+## Policies
+
+### `timescaledb.max_background_workers (int)`
+
+Max background worker processes allocated to TimescaleDB. Set to at least 1 +
+the number of databases loaded with a TimescaleDB extension in a PostgreSQL
+instance. Default value is 16.
+
+### `timescaledb.enable_tiered_reads (bool)`
+
+Enable [tiered reads][enabling-data-tiering] to that you query your data normally when it's distributed across different storage tiers.
+Your hypertable is spread across the tiers, so queries and `JOIN`s work and fetch the same data as usual.
+
+By default, tiered data is not accessed by queries. Querying tiered data may slow down query performance
+as the data is not stored locally on Timescale's high-performance storage tier.
+
+## Hypercore features
+
+### `timescaledb.default_hypercore_use_access_method (bool)`
+
+The default value for `hypercore_use_access_method` for functions that have this parameter. This function is in `user` context, meaning that any user can set it for the session. The default value is `false`.
+
+<EarlyAccess />
+
+## $SERVICE_LONG tuning
+
+### `timescaledb.disable_load (bool)`
+
+Disable the loading of the actual extension
+
+### `timescaledb.enable_cagg_reorder_groupby (bool)`
+Enable group by reordering
+
+### `timescaledb.enable_chunk_append (bool)`
+Enable chunk append node
+
+### `timescaledb.enable_constraint_aware_append (bool)`
+Enable constraint-aware append scans
+
+### `timescaledb.enable_constraint_exclusion (bool)`
+Enable constraint exclusion
+
+### `timescaledb.enable_job_execution_logging (bool)`
+Enable job execution logging
+
+### `timescaledb.enable_optimizations (bool)`
+Enable TimescaleDB query optimizations
+
+### `timescaledb.enable_ordered_append (bool)`
+Enable ordered append scans
+
+### `timescaledb.enable_parallel_chunk_append (bool)`
+Enable parallel chunk append node
+
+### `timescaledb.enable_runtime_exclusion (bool)`
+Enable runtime chunk exclusion
+
+### `timescaledb.enable_tiered_reads (bool)`
+
+Enable [tiered reads][enabling-data-tiering] to that you query your data normally when it's distributed across different storage tiers.
+Your hypertable is spread across the tiers, so queries and `JOIN`s work and fetch the same data as usual.
+
+By default, tiered data is not accessed by queries. Querying tiered data may slow down query performance
+as the data is not stored locally on Timescale's high-performance storage tier.
+
+
+### `timescaledb.enable_transparent_decompression (bool)`
+Enable transparent decompression
+
+
+### `timescaledb.restoring (bool)`
```

Comment:
This is not what restoring does. Restoring disables our internal hooks/machinery to allow things without us interfering. e.g. Restore a dump

Reply:
Added a line to explain

Reply:
restoring does not stop background workers, seems like you cpied that description from the stop_background_workers function. `restoring` disables parts of our extension to not interfere with other operations but it does not stop background workers